docs/query-layer/http.md

```
@@ -0,0 +1,521 @@
+id: 019f53ab-e706-7ae3-8281-2614077a7e83
+slug: specs/query-layer/http
+title: "Query layer — HTTP, GraphiQL, and live transport"
+type: spec
+status: active
+priority: high
+tags: [graphql, query-layer, spec]
+---
+
+# HTTP, realtime, and command mutations
+
+### Service integration: `Service::with_graphql`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)